assets/sass/patterns/molecules/main-menu.scss

```diff
@@ -5,47 +5,58 @@
 }
 
 .main-menu__title {
-  @include section-title-typeg();
   @include nospace();
   @include padding(5, "bottom");
+  @include section-title-typeg();
   text-transform: uppercase;
 }
 
 .main-menu__list {
   list-style: none;
+  @include margin(auto, "left");
+  @include margin(auto, "right");
   @include nospace();
-  margin-left: auto;
-  margin-right: auto;
+  @include padding(24, "left");
+
+  & :nth-child(5) {
+    border-bottom: 1px solid $color-text-dividers;
+  }
+
+  & :nth-child(8) {
+    border-top: 1px solid $color-text-dividers;
+  }
 }
 
 .main-menu__list_item {
+  @include blg-spacing("bottom", "extra-small");
   @include dismissible_navigation();
-  border-top: 1px solid $color-text-dividers;
-  display: block;
   @include padding($blg-space-extra-small-in-px - 1, "top");
-  padding-right: 0;
-  @include blg-spacing("bottom", "extra-small");
-  padding-left: 0;
+  @include padding(0, "right");
+  @include padding(0, "left");
+  display: block;
 }
 
 .main-menu__list_link {
+  @include font-size-and-vertical-height(14, 24);
   color: $color-text;
+  font-weight: bold;
   text-decoration: none;
+  width: 41px;
```

Comment:
The width of an inline element is ignored, so you do not need that width, please, remove it

Reply:
I didn't need it, I removed it